app/views/candidate_interface/apply_from_find/apply_on_ucas_or_apply.html.erb

```diff
@@ -1,51 +1,53 @@
-    </p>
-
-    <h2 class="govuk-heading-l">Is this new service right for you?</h2>
-    <p class="govuk-body">
-      <%= service_name %> is still in development. The service will eventually
-      replace UCAS as the way candidates apply for teacher training. However,
-      for now, it is limited to just a few courses.
-    </p>
-    <p class="govuk-body">
-      You can <%= govuk_link_to 'preview a list of training providers and courses', candidate_interface_providers_path %> currently available on <%= service_name %>.
-    </p>
-    <p class="govuk-body">
-      For most courses, you’ll still need to apply through UCAS.
-    </p>
-    <p class="govuk-body">
-      Additionally, the service isn’t ready yet for international candidates or
-      candidates who’ve studied outside the UK.
-    </p>
-
-    <p class="govuk-body">
-      <%= govuk_button_link_to t('apply_from_find.apply_button'), candidate_interface_eligibility_path(providerCode: params[:providerCode], courseCode: params[:courseCode]), class: 'govuk-!-margin-top-4 govuk-!-margin-bottom-3' %>
-    </p>
-
-    <p class="govuk-body">or</p>
-
-    <p class="govuk-body">
-      <%= govuk_link_to t('apply_from_find.ucas_apply_button'), UCAS.apply_url %>
-    </p>
+    <%= render CandidateInterface::UcasDowntimeComponent.new %>
+
+    <%= form_with model: @apply_on_ucas_or_apply, url: candidate_interface_apply_from_find_path, method: :post do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl"><%= @course.name_and_code %></span>
+          Apply for this course
+      </h1>
+
+      <p class="govuk-body">
+        The course you’ve chosen is available through a new GOV.UK service called <%= service_name %>.
+      </p>
+
+      <p class="govuk-body">
+        This will eventually replace UCAS as the way candidates apply for all teacher training.
+        However, for now, the service is limited to certain providers.
+      </p>
+
+      <p class="govuk-body">
+        <%= govuk_link_to 'See a list of available training providers and courses', candidate_interface_providers_path %>.
+      </p>
+
+      <div class="govuk-!-margin-top-6">
+        <%= f.hidden_field :provider_code, value: @course.provider.code %>
+        <%= f.hidden_field :course_code, value: @course.code %>
+
+        <%= f.govuk_radio_buttons_fieldset :service, legend: { size: 'm', text: 'Do you want to apply using a new GOV.UK service?' } do %>
+          <%= f.govuk_radio_button :service, :apply, label: { text: 'Yes, I want to apply using the new service', size: 's' }, link_errors: true, hint_text:
+            "<p class=\"govuk-body govuk-!-margin-top-2\">Choose this option if:</p>
+            <ul class=\"govuk-list govuk-list--bullet\">
+              <li>all your chosen providers are available on the new service</li>
+              <li>you want to try a streamlined service with personalised support</li>
+            </ul>".html_safe %>
```

Comment:
![image](https://user-images.githubusercontent.com/42515961/77530435-3ced9700-6e89-11ea-9de6-89c38043ffa6.png)

Just had a little play and this is an option if you would prefer it.

Reply:
`      <%= f.govuk_radio_button :service, :apply, label: { text: 'Yes, I want to apply using the new service', size: 's' }, link_errors: true %>
            <p class="govuk-body govuk-!-margin-top- govuk-!-margin-left-8">Choose this option if:</p>

            <ul class= "govuk-list govuk-list--bullet govuk-body govuk-!-margin-top-2 govuk-!-margin-left-8">
              <li>all your chosen providers are available on the new service</li>
              <li>you want to try a streamlined service with personalised support</li>
            </ul>`

Reply:
Ah thanks for this! However, although this looks the same, there would be an accessibility issue for screen readers. Because that text is not part of the `govuk-radios__item` and a `govuk-hint govuk-radios__hint`, it won't get read out when a user is on that radio button. So unfortunately going to have to keep the weird HTML safe implementation. :/